README.md

```diff
@@ -276,12 +276,12 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 > 
 >     1     
  
-**PC_Envelope(p pcpatch)** returns **bytea**
+**PC_Envelope_AsBinary(p pcpatch)** returns **bytea**
```

Comment:
It feels weird to remove PC_Envelope_AsBinary and keep PC_AsBinary, which is more or less the same  case. Should we remove it as well?